conda-recipe/meta.yaml

```diff
@@ -38,7 +38,6 @@ about:
   home: https://github.com/slaclab/lume-epics
   license: SLAC Open
   license_family: OTHER
-  license_file: LICENSE.md
```

Comment:
it should be just `LICENSE` so it matches the filename on the repo.

Reply:
Thank you for pointing this out